Add info about remotely-hosted code to 'What's new'. (#7039)
GoogleChrome/developer.chrome.com: site/en/docs/extensions/whatsnew/index.md

Check this page often to learn about changes to Chrome extensions, extensions documentation, or related policy or other changes. You'll find other notices posted on the [Extensions Google Group](https://groups.google.com/a/chromium.org/g/chromium-extensions). The [Extensions News](/tags/extensions-news/) tag lists articles about some of the topics listed here. (It even has [an RSS feed](/feeds/extensions-news.xml).) The [Chrome schedule](https://chromiumdash.appspot.com/schedule) lists stable and beta release dates.

### Guidance around remotely-hosted code {: #remotely-hosted-code }

<p class="color-secondary-text type--caption">Posted on <time>August 11, 2023</time></p>

> A requirement for Manifest V3 is that extensions may no longer use remotely-hosted code. Although this has been [part of our migration guide](/docs/extensions/migrating/improve-security/#remove-remote-code) from the beginning, we thought it was worth improving the guidance around this issue. An [addition to the migration gude]() provides more information, describing what's still possible in Manifest V3 and provide more inforamtion on strategies for upgrading.

Reviewer: I'd prefer to get rid of the "has been part of our guidance since the beginning", since we published the migration guide fairly late and it seems like an unnecessary can of worms to open (also fixed some typos and a missing link):

```suggestion
A requirement for Manifest V3 is that extensions may no longer use remotely-hosted code. We want to improve the guidance around this issue, and have made [additions to the migration guide](docs/extensions/migrating/improve-security/) to provide more information, describing what's still possible in Manifest V3 and more information on strategies for upgrading.
```

> There is a related addition to the [Troubleshooting Chrome Web Store violations](/docs/webstore/troubleshooting/). A new section describes common reasons that extensions with remotely-hosted code are rejected.

Author: ```suggestion
There is a related addition to [Troubleshooting Chrome Web Store violations](/docs/webstore/troubleshooting/). A new section describes common reasons that extensions with remotely-hosted code are rejected.
```

> There is a related addition to the [Troubleshooting Chrome Web Store violations](/docs/webstore/troubleshooting/). A new section describes common reasons that extensions with remotely-hosted code are rejected.

Author: I only did half the edit I intended here. Sorry about that. Please see the new line 24.
